apim/3.x/values.yaml

```diff
@@ -547,7 +547,7 @@ api:
         - apim.example.com
       annotations:
         kubernetes.io/ingress.class: nginx
-        nginx.ingress.kubernetes.io/configuration-snippet: "etag on;\nproxy_pass_header ETag;\nproxy_set_header if-match \"\";\n"
+        nginx.ingress.kubernetes.io/configuration-snippet: "etag on;\nproxy_pass_header ETag;\nproxy_pass_header if-match;\n"
```

Comment:
@ytvnr do you really we need if for portal API ?

Reply:
since we can configure cors for portal and set allowed header list I guess it's preferable to pass the original header.
but I maybe wrong
wdyt ?

https://github.com/gravitee-io/gravitee-api-management/blob/724f821317198ad36f151be3e4caeaafd6fdca6d/gravitee-apim-rest-api/gravitee-apim-rest-api-portal/gravitee-apim-rest-api-portal-security/src/main/java/io/gravitee/rest/api/portal/security/config/GraviteeCorsConfiguration.java#L37

Reply:
Yes, but usage of if-match is not relative to the cors itself. it is only used for checking that there is no concurrent changes to an API based on ETag. So I would say that this one is useless.

Reply:
understood! 
rolled back :)